include/kernel-defaults.mk

```diff
@@ -127,6 +127,7 @@ endef
 
 define Kernel/CompileModules/Default
 	rm -f $(LINUX_DIR)/vmlinux $(LINUX_DIR)/System.map
+	+$(KERNEL_MAKE) olddefconfig
```

Comment:
I do not like this change. 

What kernel module options are not set correctly when you do not have this?